packages/webpack/src/plugin.js

```diff
@@ -228,11 +233,38 @@ class LavaMoatPlugin {
           JAVASCRIPT_MODULE_TYPE_ESM,
         ]
 
+        // Webpack has a concept of ignored modules
+        // When a module is ignored a carveout is necessary in policy enforcement for it because the ID that webpack creates for it is not exactly helpful.
+        // example outcome in the bundle: `const nodeCrypto = __webpack_require__(/*! crypto */ "?0b7d");`
+        // Sadly, even treeshaking doesn't eliminate that module. It's left there and failing to work when reached by runtime policy enforcement.
+        // Below is the most reliable way I've found to date to identify ignored modules.
```

Comment:
(suggestion) move this into the docstring for `isIgnoredModule` under a `@remarks` tag, or something

Reply:
*grumble* the comment contains a `*/` in example.